roles/base/templates/run-ansible-pull.sh.j2

```diff
@@ -19,7 +19,17 @@ while [[ $RETRY -lt $RETRIES ]]; do
     # run ansible-pull from the directory containing the repository (and, thus, the inventory file)
     cd /home/{{ service_account}}/repo
     set +e
-    ansible-pull \
+    # if coreutils is not yet installed, we should use ansible pull alone
+    ANSIBLE_PULL=ansible-pull
+    if command -v timelimit >/dev/null
+    then
+        # 2500s ~= 40min
+        ANSIBLE_PULL="timelimit -t 2500 -T 120 $ANSIBLE_PULL"
+    else if command -v timeout >/dev/null
```

Comment:
Could you flip this? If I read the recent email from you, timeout(1) is in 10.3 but not 10.1. That means once the hosts do get upgraded then timeout will be available. I would like to prefer that once it's available in FreeBSD.